### PR TITLE
Enable more packages on clangarm64

### PR DESCRIPTION
**Pull request by driver1998**

### CODE REVIEW

File: mingw-w64-ladspa-sdk/PKGBUILD [hunk 1]
```diff
@@ -7,11 +7,11 @@ pkgver=1.17
 pkgrel=1
 pkgdesc="Linux Audio Developer's Simple Plugin API (LADSPA) SDK (mingw-w64)"
 arch=('any')
-mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
 url="https://www.ladspa.org/"
 license=("LGPL")
 source=("https://ladspa.org/download/ladspa_sdk_${pkgver}.tgz")
-sha256sums=('d9d596171d93f9c226fcdb7e27c6f917422ac487efe2c05e0a18094df4268061')
+sha256sums=('27d24f279e4b81bd17ecbdcc38e4c42991bb388826c0b200067ce0eb59d3da5b')
```
Comment (driver1998) [inline mingw-w64-ladspa-sdk/PKGBUILD:14]: Upstream seemed to repackage the same release at some point, see https://github.com/Homebrew/homebrew-core/issues/100346

**Comment by Biswa96** [comment]
mingw-w64-vo-amrwbenc failed in autobuild with some weird libtool error. But I can cross compile it in my system.

**Comment by driver1998** [comment]
Building works on my 8cx laptop as well, not sure what happened in autobuild.

**Comment by Biswa96** [comment]
Wondering if `autoreconf -fiv` may help. Generally, it is added for clang environments but to create a shared library only.

**Comment by lazka** [comment]
Here is what google gives me, pointing to lto, but not sure...: https://wiki.freebsd.org/KubilayKocak/Ports/LinkTimeOptimization#libtool:_eval:_1:_Syntax_error:_.22.7C.22_unexpected

**Comment by lazka** [comment]
> Wondering if `autoreconf -fiv` may help. Generally, it is added for clang environments but to create a shared library only.
> 
> Though that command sometimes break with old autotools files.

can't hurt

**Comment by lazka** [comment]
it worked :)

**Pull request merged**